CMakeLists.txt

```diff
@@ -62,7 +69,25 @@ else()
     cxx_lambda_init_captures
     cxx_generic_lambdas
   )
+  set(QUICKCPPLIB_REQUIRE_CXX_STANDARD 201402L)
+endif()
+
+if(QUICKCPPLIB_USE_SYSTEM_BYTE_LITE)
+  find_package(byte-lite REQUIRED)
+  all_link_libraries(PUBLIC nonstd::byte-lite)
```

Comment:
How do we know the package manager has named the target `nonstd::byte-lite`? Because sometimes they rename them (which I agree is highly irritating).

I think this also needs to be an overridable option, plus the package name too as `byte-lite` may also have been renamed. Same goes for all the other dependencies.

Reply:
I'd argue that if a package manager chose to patch the buildsystem of a `*-lite` library to change its package and/or target name, it would be the package managers responsibility to patch our build system to be compatible.

In any case I'd at least defer implementing this until someone actually requests such configuration options.